test/integration/index.js

```diff
@@ -297,10 +312,15 @@ describe( 'app integration ->', function() {
         } );
 
 
-        it( 'should return a 404', function( done ) {
+        it( 'should return a placeholder image', function( done ) {
           req = http.request( reqOptions, function( res ) {
-            expect( res.statusCode ).to.be.equal( 404 );
-            done();
+            expect( res.headers[ 'content-type' ] ).to.be.equal( 'image/jpeg' );
+
+            res.on( 'data', function( chunk ) {
+              expect( JSON.stringify( chunk ) ).to.be.equal( JSON.stringify( placeholderImage ) );
+
```

Comment:
if the content type is image/jpeg, then the data chunk will be a block out of a binary image file, so what's the purpose in stringifying it before doing the comparison? (that's actually a question, not just a veiled way to say "don't stringify", although if there isn't a good reason, then don't :wink:)